docs/docs/en/guides/develop/projects/plugins.md

```diff
@@ -21,8 +21,8 @@ Note that plugins are designed to be a simple way to extend Tuist's functionalit
 
 If you need more flexibility, consider suggesting a feature for the tool or building your own solution upon Tuist's generation framework, [`TuistGenerator`](https://github.com/tuist/tuist/tree/main/Sources/TuistGenerator).
 
-> [!WARNING] NON-ACTIVELY-MAINTAINED
-> Our plugin infrastructure is not actively maintained. We are looking for contributors to help us improve it. If you are interested, please reach out to us on [Slack](https://slack.tuist.io/).
+> [!WARNING] DEPRECATED
+> Plugins are deprecated and we are working on a more flexible and powerful solution, <LocalizedLink href="/en/guides/develop/automate/workflows">workflows</LocalizedLink>. We recommend not developing new plugins until the new solution is available.
```

Comment:
This is not entirely correct – `ProjectDescriptionHelpers` plugins are here to stay, at least for the time being? It's only tasks that should be deprecated.